tests/messagebox/execution.livecodescript

```diff
@@ -60,17 +60,36 @@ on TestBug15832
 	TestAssert "; in single line message box", word 1 of msg is "field"
 end TestBug15832
 
+on TestBug16863
+   local tStack
+   create stack
+   put the short name of it into tStack
+   
+   ## Create a button on the stack
+   set the defaultStack to tStack
+   create button
+   set the name of it to "test"
+   
+   local tValidScript, tScript
+   put "there is a button" && quote & "test" & quote into tScript
+   
+   ideExecuteScript "there is a button" && quote & "test" & quote, tStack, empty, true, tValidScript
+   TestAssert "boolean expression with 'put' prepended",  msg is "true"
```

Comment:
Can you delete this line? It's my fault because of the wording of my message! It's just the same test as the one below.